### ISSUE TITLE
Justification for Permissions?

### ISSUE THREAD

**Issue by lucidBrot** [everyonesdesign/disable-google-search-text-highlights#1]
Hi! It's very kind of you that you published this extension. <3

I have just rated it 5 stars on the extension store, but I wonder why it requires permission to read and change content on websites. Would it be possible to only require access to the URL instead?

Also, do you have a paypal or such where I could drop you money for a coffee?

**Comment by everyonesdesign** [comment]
Hello!

Thank you for raising this concern. The context about this decision is that I wanted the extension to work on all the Google domains, and keep it as maintanance-free as possible (current implementation uses a generic regexp to detect a Google domain).

It is possible to list all the Google domains (I used [this link](https://ipfs.io/ipfs/QmXoypizjW3WknFiJnKLwHCnL72vedxjQkDDP1mXWo6uco/wiki/List_of_Google_domains.html)) in the permissions of the extension, but I found the approach with listing all the domains explicitly as a more fragile one, since if Google adds a new domain, it won't work for it.

Update: I checked it now, and found this official Google link: [supported domains](https://www.google.com/supported_domains). This is pretty awesome and easy to monitor. I'll look into migrating the extension to this list and will create a watcher that notifies me about the changes of this page, so I can adjust the permissions in time. 

This issue will be kept open unless the migration is complete.

> Also, do you have a paypal or such where I could drop you money for a coffee?

Thanks! 🙂 Now I don't have it a link to share for that, but maybe later I'll add something like it to the description of the extension.
